src/app/firedancer/topology.c

```diff
@@ -956,7 +956,8 @@ fd_topo_configure_tile( fd_topo_tile_t * tile,
 
   } else if( FD_UNLIKELY( !strcmp( tile->name, "snapin" ) ) ) {
 
-    tile->snapin.funk_obj_id = fd_pod_query_ulong( config->topo.props, "funk", ULONG_MAX );
+    tile->snapin.funk_obj_id   = fd_pod_query_ulong( config->topo.props, "funk",     ULONG_MAX );
+    tile->snapin.tcache_obj_id = fd_pod_query_ulong( config->topo.props, "txncache", ULONG_MAX );
```

Comment:
```suggestion
    tile->snapin.txncache_obj_id = fd_pod_query_ulong( config->topo.props, "txncache", ULONG_MAX );
```

tcache has a different meaning in fd